torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -914,7 +933,14 @@ void CudaCodeGen::Initialize() {
   }
 
   std::string func_name = GetUniqueFuncName("func");
-  os() << "extern \"C\" __global__" << std::endl << "void " << func_name << "(";
+  os() << "extern \"C\" __global__" << std::endl;
+#ifdef USE_ROCM
+  // TODO: If the work group can be larger than 256, we have to annotate
```

Comment:
Please give the constant 128 and 1024 a name here. So people know its meaning. 

Also if possible, please include a reference/link where they come from, so future developers know how to update them. Thanks!

Reply:
Would describing them in the comment more explicitly work or do you prefer a `#define` or somesuch?

Reply:
I would prefer a "static const int kBlcokSizeLimit..." or something like that. I think having both comments and variable names will be even more helpful for future developers. But it is up to you. Thanks!

Reply:
I think with the new commentary, it should work. It's a much better comment now, thank you for suggesting that it needed improvement.